IcandoPRs/mridal

```diff
@@ -0,0 +1 @@
+Hello
```

Comment:
This is a bit to relaxed! More formal please!

Reply:
Ok, change to "Dear sir/madame"